exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/lateraljoin/TestE2EUnnestAndLateral.java

```diff
@@ -394,4 +401,30 @@ public void testLateral_HashAgg_with_nulls() throws Exception {
       .baselineValues("dd",222L)
       .build().run();
   }
+
+  @Test
+  public void testUnnestTopN() throws Exception {
```

Comment:
Please make this test part of `TestLateralPlans.java` where we have other plan verification tests.

Reply:
Thanks, done.